bindings/python/pyproject.toml

```diff
@@ -26,7 +26,7 @@ dynamic = [
     'license',
     'readme',
 ]
-dependencies = ["huggingface_hub>=0.16.4,<0.17"]
+dependencies = ["huggingface_hub>=0.16.4,<1.0"]
```

Comment:
No, we need to allow for breaking changes in huggingface_hub meaning we can do `>=0.17,<0.18` instead.